tests/testDipoleFitter.py

```diff
@@ -226,51 +209,23 @@ def testDipoleTask(self):
                              result2['ip_diffim_PsfDipoleFlux_neg_centroid_y'],
                              rtol=0.01)
 
-            if self.params.display:
-                self.testImage.displayCutouts(r1)
-        if self.params.display:
+            if params.display:
+                params.testImage.displayCutouts(r1)
+        if params.display:
             plt = ipUtils.importMatplotlib()
             if not plt:
                 return result
             plt.show()
 
         return result
 
-
-class DipoleFitTaskEdgeTest(DipoleFitTaskTest):
-    """ A test case to test the task in the same way as the algorithm:
-    Here test that dipoles too close to the edge are raised correctly.
-    """
-    def setUp(self):
-        # Ensure that both dipoles will fail (too close to edge)
-        self.params = DipoleFitTestGlobalParams()
-        self.params.xc = np.array([5.3, 2.2])  # xcenters of two dipoles in image
-        self.params.yc = np.array([2.6, 98.5])  # ycenters of two dipoles
-
-        offsets = self.params.offsets
-        self.testImage = ipUtils.DipoleTestImage(
-            xcenPos=self.params.xc + offsets,
-            ycenPos=self.params.yc + offsets,
-            xcenNeg=self.params.xc - offsets,
-            ycenNeg=self.params.yc - offsets,
-            flux=self.params.flux, fluxNeg=self.params.flux,
-            gradientParams=self.params.gradientParams)
-
-        self.catalog = self.testImage.detectDipoleSources()
-
-    def tearDown(self):
-        DipoleFitTaskTest.tearDown(self)
-
-    def testDipoleTask(self):
-        pass
-
     def testDipoleEdge(self):
         """Test the dipole fitting singleFramePlugin. Test that the dipoles
         which are too close to the edge raise the correct exception.
```

Comment:
There's no testing of exceptions in this test. Did you neglect a "with self.assertRaises"?